### ISSUE TITLE
Comparison with tensorflow1/2 implementation

### ISSUE THREAD

**Issue by a-z-e-r-i-l-a** [DLR-RM/stable-baselines3#56]
I wanted to ask if you expect better performance using pytorch in stable-baselines3 than if it was going to be implemented with tensorflow2 ( or compared to the currently implemented tensorflow1). If so, from what aspects do you expect the advantages be in the end?

**Comment by a-z-e-r-i-l-a** [comment]
yea I meant about wall clock time or any other aspect which you may have thought about, specifically compared to TF2. 

It seemed to me that there is a tendency towards PyTorch more than TF2 but I couldn't guess why. and was wondering if there is any advantage in using PyTorch compared to TF2.

**Comment by araffin** [comment]
i guess reading those two issues will give you the answer then:
- https://github.com/hill-a/stable-baselines/issues/733
- https://github.com/hill-a/stable-baselines/issues/576
